src/renderers/shared/stack/reconciler/ReactChildReconciler.js

```diff
@@ -127,6 +130,13 @@ var ReactChildReconciler = {
         // The child must be instantiated before it's mounted.
         var nextChildInstance = instantiateReactComponent(nextElement);
         nextChildren[name] = nextChildInstance;
+        mountImages[name] = ReactReconciler.mountComponent(
```

Comment:
We use an array for this in `mountChildren` so I can change it here too if necessary.